config/default/kustomization.yaml

```diff
@@ -28,7 +28,7 @@ patchesStrategicMerge:
   # Protect the /metrics endpoint by putting it behind auth.
   # If you want your controller-manager to expose the /metrics
   # endpoint w/o any authn/z, please comment the following line.
-- manager_auth_proxy_patch.yaml
+#- manager_auth_proxy_patch.yaml
```

Comment:
Is this an intended change to disable authn/z ?

Reply:
yes, as most ppl don't have it setup :D